opcua/common/manage_nodes.py

```diff
@@ -85,7 +85,13 @@ def create_variable(parent, *args):
     """
```

Comment:
@oroulet OK.

@bitkeeper plase check discussion in #236 regarding the semantics of functions. So it seams good to add this functions with *_write* sufix